i18n/fr/docusaurus-plugin-content-docs/version-22.10/monitoring/basic-objects/hosts.md

```diff
@@ -13,127 +13,133 @@ Pour créer un hôte manuellement :
 
 ### Information de base sur l'hôte
 
-* Le champ **Nom** définit le nom d’hôte qui sera utilisé par le moteur de supervision. Les caractères suivants ne sont pas autorisés : `~!$%^&|'"<>?,()=*{}` et les espaces.
-* Le champ **Alias** indique l’alias de l’hôte.
-* Le champ **Adresse** définit l'adresse IP ou le nom DNS de l’hôte. Le bouton **Résoudre** permet de résoudre le nom de
-  domaine en interrogeant le serveur DNS configuré sur le serveur central.
-* Les champs **Communauté SNMP & Version** contiennent respectivement le nom de la communauté ainsi que la version SNMP.
-* Le champ **Serveur de supervision** indique quel est le serveur de supervision chargé de superviser cet hôte.
-* Le champ **Fuseau horaire** indique l'emplacement du fuseau horaire des hôtes surveillés.
-* Le champ **Modèles** permet d’associer un ou plusieurs modèles d’hôtes à cet objet.
+* **Nom** : nom d’hôte utilisé par le moteur de supervision. Les caractères suivants ne sont pas autorisés : `~!$%^&|'"<>?,()=*{}`. De plus, les espaces seront remplacés par des `_`.
+* **Alias** : un autre nom pour l'hôte. Les espaces et caractères interdits dans le nom peuvent être employés ici. L'alias peut être utilisé dans la **Barre de recherche** de la page **Statut des ressources** avec la syntaxe `alias:` pour retrouver un hôte.
```

Comment:
It would be preferable to use `h.alias:` instead of `alias:`.
And what about `h.name:` and `h.address:` for **Name** and **Address**?